SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -3484,6 +3484,24 @@ \section{Writing Tests}
 in ITK.
 
 
+\subsection{Code Layout in Tests}
+\label{subsec:CodeLayoutInTests}
+
+The following general layout is recommended for ITK unit tests:
+\begin{itemize}
+\item Constant declarations (e.g. image dimensions, etc.).
+\item General typedef declarations (e.g. pixel and image types, etc.).
```

Comment:
It helps readability to have constants and typedef's local to where they are used -- we could leave these two items out.

Reply:
08df219 removes the item but adds a note after the item list. Have a look to see if that makes sense or whether I should rather drop the note and only leave the items out.